Updates/Instances/565_gruul.sql

```diff
@@ -135,15 +132,15 @@ INSERT INTO `spawn_group_spawn` (`Id`, `Guid`, `SlotId`) VALUES
 (@SGGUID+1, @CGUID+4, -1), -- Kiggler the Crazed
 (@SGGUID+1, @CGUID+5, -1), -- Blindeye the Seer
 (@SGGUID+2, @CGUID+6, -1), -- Gruul the Dragonkiller
-(@SGGUID+3, @CGUID+7, -1), -- spawn_group_entry
+(@SGGUID+3, @CGUID+7, -1), -- Lair Brute
 (@SGGUID+4, @CGUID+8, -1), -- spawn_group_entry
 (@SGGUID+4, @CGUID+9, -1), -- spawn_group_entry
-(@SGGUID+5, @CGUID+10, -1), -- spawn_group_entry
```

Comment:
give creature groups a hierarchy please, i know its possible to not do it, but it shouldn't be a thing

Reply:
you mean not using -1 for SlotId?

> -1 for gameobjects or creatures that are not in formation but part of a spawn group

Reply:
yeah

Reply:
Yeah would be possible to do so, but i dont see a reason tbh. And also like quoted above our wiki currently says to explizit use -1 for non formation groups
Guess thats something @killerwife or @AnonXS has to decide

Reply:
has nothing to do with formations, every group has a hierarchy and it shows
your laziness will result in more work for someone else in the future
ill change the wiki

Reply:
If kw says it's fine to use it like that i dont mind, i'm just doing it like that because that's how it is documented.